flake.nix

```diff
@@ -0,0 +1,36 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system};
+      in
+      with pkgs; {
+        devShell = with pkgs; mkShell {
+          buildInputs = [
+            docker-client
+            gnumake
+            envsubst
+          ];
+
+          shellHook = ''
```

Comment:
In the same spirit as to why you have it here, it might be nice to add this "quick and dirty" set of examples to the README as a pre-amble to the documentation link there.

Reply:
Done